recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -32,9 +32,17 @@ PACKAGECONFIG ?= " \
     release \
     ${@bb.utils.contains('DISTRO_FEATURES', 'opencdm', 'opencdm opencdm_gst', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_nexus_svp', 'opencdmi_prnx_svp', '', d)} \
-    compositorclient virtualinput websource webkitbrowser \
+    virtualinput websource webkitbrowser \
     "
 
+# add compositor client if Wayland is present
+PACKAGECONFIG_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'compositorclient', '', d)}"
+# add it for RPI regardless if Wayland is present (because we can do dispmanx too)
+PACKAGECONFIG_append_rpi = " ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '', 'compositorclient', d)}"
+# just not with vc4graphics
+PACKAGECONFIG_remove = "${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', 'compositorclient', '', d)}"
+
```

Comment:
'compositorclient' should not be removed as wpebackend-rdk have dependancy if wpeframework backend is selected

Reply:
I don't think compositorclient works with vc4graphics yet, as it has direct dependencies on bcm_host and dispmanx drivers. We'll sort that out once the compositor setup supports vc4graphics/libdrm/mesa.